packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -696,8 +698,12 @@ export const prepareVaultManagerKit = (
               );
               const vaultDebt = floorMultiplyBy(debtAmount, debtPortion);
               const collatPostDebt = AmountMath.subtract(vCollat, vaultDebt);
+              const collatPostPenalty = AmountMath.subtract(
+                collatPostDebt,
+                ceilMultiplyBy(getVaultPenalty(debtAmount), debtPortion),
```

Comment:
After I understood that `debtPortion` is the amount of "collateralPenalty per debt", the rest fell into place.